_data/simple-icons.json

```
@@ -8530,6 +8530,11 @@
             "hex": "1C3C3C",
             "source": "https://www.langchain.com"
         },
+        {
+            "title": "LanguageTool",
+            "hex": "45A1FC",
+            "source": "https://languagetool.org/images/logo_color.svg"
```

```suggestion
            "source": "https://languagetool.org"
```
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines we should link to the page the SVG is found on (footer of the home page) instead of directly to the SVG.